src/main.c

```diff
@@ -119,21 +119,21 @@ int main(int argc, char *argv[])
         }
     }
     Server *server = serverNew(server_port);
-    serverAddHandler(server, notFound);
-    serverAddStaticHandler(server);
-    serverAddHandler(server, about);
-    serverAddHandler(server, signup);
-    serverAddHandler(server, logout);
-    serverAddHandler(server, login);
-    serverAddHandler(server, search);
-    serverAddHandler(server, connect);
-    serverAddHandler(server, like);
-    serverAddHandler(server, unlike);
-    serverAddHandler(server, post);
-    serverAddHandler(server, profile);
-    serverAddHandler(server, dashboard);
-    serverAddHandler(server, home);
-    serverAddHandler(server, session);
+    serverAddRoute(server, NONE, NULL, notFound);
```

Comment:
Instead, you can split the handler in a dedicated routine as [cgi_url_dltrie.c](https://github.com/embedded2016/web-server-cgi/blob/master/src/cgi_url_dltrie.c) does.